mkdocs.yml

```diff
@@ -30,7 +30,14 @@ theme:
 nav:
   - Introduction: 'index.md'
   - 1. Introduction:
-    - Test: "./1-introduction/section-1.md"
+    - Overview: "introduction/overview.md"
+    - Project Architecture: "introduction/project-architecture.md"
+    - Toolchain:
+      - Ansible: "introduction/toolchain/ansible.md"
+      - Conda: "introduction/toolchain/conda.md"
   - 2. Installation:
-    - Overview: "./2-installation/OVERVIEW OF INSTALLATION PROCESS.md"
-    - Steps: "./2-installation/Installations step.md"
\ No newline at end of file
+    - Overview: "installation/OVERVIEW OF INSTALLATION PROCESS.md"
+    - Steps: "installation/Installations step.md"
+    - Installing miniconda: "installation/Installing Miniconda.md"
```

Comment:
is it Miniconda suppose to be Capitalize the first letter?

Reply:
Yes, from [their documentation site](https://docs.conda.io/en/latest/miniconda.html) it seems we need to capitalize it.

Reply:
Oh yes I see what you mean now, I will fix the un-capitalized page title.

Reply:
Fixed in [20d8d94](https://github.com/InTensorfiers/intensorfiers.github.io/pull/9/commits/20d8d94a88727d5850d590331258de4a24db6eda).

Reply:
Ok Thanks I will approved.